std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/LongStorageFacade.java

```diff
@@ -0,0 +1,55 @@
+package org.enso.table.data.column.storage.numeric;
+
+import java.util.function.ToLongFunction;
+import org.enso.table.data.column.storage.ColumnLongStorage;
+import org.enso.table.data.column.storage.ColumnLongStorageIterator;
+import org.enso.table.data.column.storage.ColumnStorage;
+import org.enso.table.data.column.storage.ValueIsNothingException;
+import org.enso.table.data.column.storage.type.IntegerType;
+import org.enso.table.data.column.storage.type.StorageType;
+
+/** A facade for a column storage that converts the stored type to a long. */
+public class LongStorageFacade<T> implements ColumnLongStorage {
```

Comment:
Unless you want clueless developers to subclass, then:
```suggestion
public final class LongStorageFacade<T> implements ColumnLongStorage {
```